### PR TITLE
flash.sh: esptool run => mpremote reset

### PR DESCRIPTION
**Pull request by CRImier**
working around d a probably-local-software-stack bug. tested, works the same, but doesn't require a BOOT button press in the end of the script for me. should not affect you if your esptool works correctly unlike mine.

### CODE REVIEW

File: tools/flasher/flash.sh [hunk 1]
```diff
@@ -134,7 +134,7 @@ cyan_underlined "Registering device information (5/5)"
 DEVINFO=`mpremote connect "port:$ESPTOOL_PORT" run ./register-device.py | sed "s/\r$//"`
 echo "$DEVINFO,$FIRMWARE_FILE_NAME,$AIRGUARD_VERSION" >> devices-flashed.txt
 
-esptool.py run
+mpremote reset
```
Comment (kasparsd) [inline tools/flasher/flash.sh]: Can we please add the ‘ connect "port:$ESPTOOL_PORT"’ part for this similar to the other mpremote calls to help on systems with multiple UART devices?

Comment (CRImier) [reply tools/flasher/flash.sh]: yes, but obv outside of scope of this PR

Comment (kasparsd) [reply tools/flasher/flash.sh]: The current change as is breaks the tool on all systems with multiple UART devices. We would need to add that connection config for it to continue working.

Comment (CRImier) [reply tools/flasher/flash.sh]: oh! oh misread! I see what you mean now! will check for that

Comment (CRImier) [reply tools/flasher/flash.sh]: fixed! thank you for pointing this out!

**Comment by CRImier** [comment]
updated to have `mpremote connect`, do re-review

**Review by kasparsd — APPROVED** [review]

**Pull request merged**